README.md

```diff
@@ -1,8 +1,8 @@
-# Profiles
+# GP Finder
 
-[![Coverage Status](https://coveralls.io/repos/github/nhsuk/profiles/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/profiles?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/nhsuk/gp-finder/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/gp-finder?branch=master)
 
-A collection of health service provider profiles. Initially containing GP practices and Pharmacies.
+A GP finder application. Will used for booking a GP and to find a new GP to register with.
```

Comment:
Atm, it is only going to be a finder. The booking thing will be a page within the profiles app.